ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/sflow.j2

```diff
@@ -34,4 +34,20 @@ sflow source-interface {{ sflow.source_interface }}
 {%     if sflow.run is arista.avd.defined(true) %}
 sflow run
 {%     endif %}
+{%     if sflow.hardware_acceleration.enabled is arista.avd.defined(true) %}
+sflow hardware acceleration
+{%     endif %}
+{%     if sflow.hardware_acceleration.sample is arista.avd.defined %}
+sflow hardware acceleration sample {{ sflow.hardware_acceleration.sample }}
+{%     endif %}
+{%     for module in sflow.hardware_acceleration.modules | arista.avd.natural_sort %}
```

Comment:
please add `('name')` on the natural_sort, so it sorts on the name key.

Reply:
Fixed!